client/extensions/woocommerce/app/settings/payments/payments-off-site.js

```diff
@@ -25,7 +52,17 @@ class SettingsPaymentsOffSite extends Component {
 							'information'
 						)
 					} />
-				<Card></Card>
+					<ListTable>
+						<ListHeader>
+							<ListTd width="30">Method</ListTd>
```

Comment:
Also, why not just putting the full `"30%"` here? It makes it clearer at a glance what those "widths" mean.